content/posts/2020-03-21-getting-r-to-print-english-error-messages_es.Rmarkdown

```diff
@@ -0,0 +1,67 @@
+---
+title: Cómo hacer que R devuelva errores en inglés
+author: Thomas Neitmann
+date: '2020-03-21'
+slug: getting-r-to-print-english-error-messages
```

Comment:
Could you translate the `slug` as well. This will be used as the URL of the post.

Reply:
sure, will update it